rotkehlchen/api/v1/resources.py

```diff
@@ -2075,3 +2076,20 @@ class LimitsCounterResetResource(BaseResource):
     @use_kwargs(post_schema, location='view_args')
     def post(self, location: str) -> Response:
         return self.rest_api.reset_limits_counter(location)
+
+
+class StakingResource(BaseResource):
+    get_schema = StakingQuerySchema
+
+    @use_kwargs(get_schema, location='json_and_query')
+    def get(
+            self,
+            async_query: bool,
+            only_cache: bool,
+            filter_query: StakingQuerySchema,
+    ) -> Response:
+        return self.rest_api.query_kraken_staking_events(
```

Comment:
I noticed that I forgot the kraken name here. Will change after comments. What I wanted to do with this endpoint is to provide something generic so other exchanges/protocols can be queried also using the base history entries

Reply:
I am not sure what you mean by this comment

Reply:
Nothing now. Is what about making sense between the route name and the name of the methods used